app/views/layouts/missionsystem.html.slim

```diff
@@ -0,0 +1,24 @@
+doctype html
+html
+  head
+    title
+      | Ruby on Rails Tutorial Sample App |
```

Comment:
The code is copied from the Rails Tutorial. Later I'll write my own page content